implementations/rust/ockam/ockam_transport_core/src/error.rs

```diff
@@ -38,6 +38,8 @@ pub enum TransportError {
     PortalInvalidState,
     /// InvalidRouterResponseType
     InvalidRouterResponseType,
+
+    TempError,
```

Comment:
What kind of error does this denote?

Reply:
This is me trying to catch WebSocket bug on CI, apparently it doesn't reproduce now 🥲
I'll remove it before the merge